src/test/java/core/basesyntax/handler/impl/ReturnOperationHandlerTest.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax.handler.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.basesyntax.handler.OperationHandler;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import core.basesyntax.storage.Storage;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class ReturnOperationHandlerTest {
+    private static final String ORANGE = "orange";
+    private OperationHandler handler;
+
+    @BeforeEach
+    void setUp() {
+        handler = new ReturnOperationHandler();
+        Storage.clear();
```

Comment:
better clear storage after each. 
Imagine that one of your colleagues adds a new test class and expects the storage to be empty but it contains data after performing your test.